wiki/Tournaments/IGTS/2018/en.md

```diff
@@ -0,0 +1,139 @@
+| Designer | ![][flag_US] [Swovine](/users/6666316) |
+| Statisticians | ![][flag_FR] [Kasumii-sama](/users/6177263),  ![][flag_FR] [TLQ_Yoshii](/users/7157133) |
+| Referees | ![][flag_FR] [Kasumii-sama](/users/6177263), ![][flag_FR] [TLQ_Yoshii](/users/7157133), ![][flag_AU] [cactusthegreat](/users/5592409), ![][flag_US] [ITotallyGetThat](/users/8770622) |
+| Streamer | ![][flag_DE] [Heamburger](/users/4705120) |
+| Casters | ![][flag_MY] [Bedwyr Aorta](/users/10875855), ![][flag_EC] [Gamelan4](/users/9856910) |
+
+## Links
+
+- [Discussion Thread](/community/forums/topics/767841)
+- [IGTS Discord](https://discord.gg/qavX3Ga)
+- [Livestream](https://www.twitch.tv/beginnerstaikotourney)
+- [Player/Team Registration Form](https://docs.google.com/forms/d/e/1FAIpQLScmGzj3NMAOimVUzSdMRDa8COV5MOxcANOljLuxQjrJ9wLIvg/viewform)
+- [Staff Registration Form](https://docs.google.com/forms/d/e/1FAIpQLSe39Rk4JyUQ_GGBJHdYxfA7kZmJQqboK3p1LV_eTYZf-8Td_Q/viewform)
+- [Challonge Brackets](https://challonge.com/u85xl34m)
+
+-----------------------
+
+## Participants
+
+*The participants will be listed once the registration phase is finished.*
+
+-----------------------
+
+## Mappool
+
+*The mappools will be revealed one week in advance for each stage.*
+
+------------------------------------------------------------------------
+
+## Ruleset
+
+### General Rules
+
+1. The Intermediate Global Taiko Showdown 2018 is a two-on-two Team Vs tournament, with the aspect of double-elimination bracket.
+2. In order for your registration to count, you must meet following conditions:
+   - Participant must fill out the provided form.
+   - Participant must be within rank **3,500 - 10,000 during registrations.**
+   - Participant is not allowed to rank up further than rank **2,500 during the tournament's occurrence.**
+   - Participant must join tournament's Discord server.
+3. The mapset will be announced the week before a stage begins.
+4. Match schedule is decided by the organisers, which will announce schedule the same day as the mapset every week. Each stage of this tournament will last a week, and that week starts when the mapset is announced a day before, so players need to get their match in before the week is over.
+5. If both teams get a tie on the TieBreaker, the TieBreaker will be replayed with FreeMod applied.
+6. If a player disconnects from the game, the opponent wins the round. 
+   - If the player disconnects during the first five seconds of the song, the match will be restarted.
+7. Maps cannot be used twice in the same round.
+8. If a player does not show up within **15 minutes** of the start time, the enemy wins by default.
+9. Any other rules, changes, or unexpected occurrences will be announced in the tournament's forum thread.
+
+### Stage Regulations
+
+1. There are four stages to this tournament: Group Stage, Quarterfinals, Semifinals, and Finals.
```

Comment:
Looking below, isnt this like five stages? (Round of 16 is missing)

Reply:
Right, the tournament host's original schedule was so confusing lol. I'll fix asap.